mod_ci/controllers.py

```diff
@@ -313,7 +315,7 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
         #     return
         # TODO: check what happens on merge conflicts
     else:
-        test_branch = repo.create_head('CI_Branch', 'HEAD')
+        test_branch = repo.create_head('CI_Branch_' + kvm_name, 'HEAD')
```

Comment:
This method won't work. We don't need separate branches (the VM's don't care about git), but separate directories with separate git clones.